Simple_Prep_docker

```diff
@@ -66,11 +66,9 @@ sed-i "$dockerfile" -e '/DL_APT/d'
 tag_=simple_prep_${USER}_${DL_DIST}_${DL_DATE}
 tag=repronim:${tag_}
 echo "I: tag $tag"
-if docker images | grep -q repronim.*${tag_}; then
-    echo "I: tag $tag already exists -- skipping rebuilding"
-else
+if [ $(command -v docker) ]; then
     docker build -t $tag -f $dockerfile .
```

Comment:
wouldn't it build and only then fail if tag already exists? (sorry didn't try, but I thought that my above check was to fail early for that reason)